src/main/java/lab3/swingMain.java

```diff
@@ -0,0 +1,60 @@
+package lab3;
+
+import lab3.game.Board;
+import lab3.ui.swingConsole;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class swingMain {
+    /**
+     * This is Not Fully-Functional
+     */
+    public static void main(String[] args) {
+        Board board = new Board();
```

Comment:
A good attempt, but you have some design issues to clean up that would make implementing this simpler.  See the sample solution.